Andman/Havelock/my_adventure

```diff
@@ -0,0 +1,3 @@
+Awesome scuba diving in havelock at beach number 2.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.